guides/source/action_controller_overview.md

```diff
@@ -374,11 +374,14 @@ own code to adapt to your situation.
 Session
 -------
 
-Your application has a session for each user in which you can store small amounts of data that will be persisted between requests. The session is only available in the controller and the view and can use one of several of different storage mechanisms:
+Your application has a session for each user in which you can store small
+amounts of data that will be persisted between requests. The session is only
+available in the controller and the view. The storage mechanism can be
+configured to a store provided by an external gem or one that Rails provides by
+default:
```

Comment:
Instead of changing this part, what if we added a mention of third-party gems as a bullet point to the list below? Maybe something like:

```
* A custom store or a store provided by a third-party gem. 
```

Reply:
![image](https://user-images.githubusercontent.com/34329697/174456928-ad772150-5fa7-4e43-983f-2f46a78fd6a7.jpeg)

Reply:
https://docs.google.com/forms/d/1gyJmYaaKiEUbv80kwROtapt22SjDEVlJQe87xgwlk94/edit